yarn-project/noir-contracts/src/libs/custom-notes/src/transparent_note.nr

```diff
@@ -64,8 +64,9 @@ impl TransparentNote {
     fn emit_nullifier(_self: Self, mut context: Context, secret: Field, siloed_commitment: Field, index: Field) -> Context {
         // Create a nullifier for the message based on its index in the tree
         
+        // TODO: we're coupling nullifier with siloed_commitment, is that okay?
```

Comment:
This is probably the correct thing to do, but confirm from David or Jean.

Reply:
Hmm is the commitment siloed with contract address here? Or just with slot? I think we do NOT want to silo it with contract address before outputting it from app circuit as that is the kernels job.

Reply:
Good point. This seems to be just the "commitment" before any contract address siloing.